frontend/src/metabase/components/form/FormField.jsx

```diff
@@ -62,6 +62,7 @@ function FormField(props) {
     hidden = formField && (formField.hidden || formField.type === "hidden"),
     horizontal = formField &&
       (formField.horizontal || formField.type === "boolean"),
+    isToggle = formField && formField.type === "boolean",
```

Comment:
It seems it should just be a local variable, rather than a prop. If it's a prop, I'll be able to set `isToggle={true}` on any other input type and it'd be confusing